README.md

```diff
@@ -122,22 +179,134 @@ Supported Backends:
 Cryptofeed supports some REST interfaces for retrieving historical data and placing orders. See the [rest](https://github.com/bmoscon/cryptofeed/tree/master/cryptofeed/rest) package.
 
 
-# Planned Work
+## Planned Work
 
-## Future Feeds
+### Future Feeds
 * CEX
 * BTCC
 * Many more...
 
-## REST
+### REST
 Continue to build out rest endpoints and standardize exchange interfaces and data
 
-## Additional Callback Methods / Backends
+### Additional Callback Methods / Backends
 * Pulsar
 * More ZMQ improvements/options
 
-# Contributing
+## Contributing
 Issues and PRs are welcomed. If you'd like to discuss ongoing development please join the [slack](https://join.slack.com/t/cryptofeed-dev/shared_invite/enQtNjY4ODIwODA1MzQ3LTIzMzY3Y2YxMGVhNmQ4YzFhYTc3ODU1MjQ5MDdmY2QyZjdhMGU5ZDFhZDlmMmYzOTUzOTdkYTZiOGUwNGIzYTk)
 
 This wouldn't have been possible with the many [contributors](AUTHORS.md)! I owe them and all who have contributed in other ways my thanks!
 
+## Pipenv
```

Comment:
Not sure this belongs in the readme, if at all. If someone uses pipenv already, they'll know how to do all this.

Reply:
My idea was to help the developers who do not know Pipenv yet.
For this reason I gave several command line examples.
We can move that chapter at the end of the `Pipfile` if you prefer.

Reply:
Maybe move this to the install file?